scripts/ci-tool.py

```diff
@@ -114,7 +116,7 @@ def create_junit_format(output_path, data):
                     testcase,
                     'failure',
                     type=status,
-                    message='Test {}'.format(status))
+                    message=f'Test {status}')
```

Comment:
Not sure everyone is running 3.7 yet. For compat. Let’s use previous syntax

Reply:
But f-strings are so purdy... Done.

Reply:
Is it not the assumption the ci-tools are ran in docker by default?

Reply:
You are right that we should make sure the docker image is using the lowest version available. We can add that too the list